src/plays/tic-tac-toe/Game.jsx

```diff
@@ -0,0 +1,160 @@
+          }}
+        >
+          Play Again!
+        </button>
+      </div>
+    );
+  };
+
+  // Game Logic
+  const findWinner = () => {
+    if (
+      gameArray[0] === gameArray[1] &&
+      gameArray[0] === gameArray[2] &&
+      gameArray[0] !== ""
+    ) {
+      setWinMessage(`${gameArray[0]} has won!`);
+    } else if (
+      gameArray[3] === gameArray[4] &&
+      gameArray[3] === gameArray[5] &&
+      gameArray[3] !== ""
+    ) {
+      setWinMessage(`${gameArray[3]} has won!`);
+    } else if (
+      gameArray[6] === gameArray[7] &&
+      gameArray[6] === gameArray[8] &&
+      gameArray[6] !== ""
+    ) {
+      setWinMessage(`${gameArray[6]} has won!`);
+    } else if (
+      gameArray[0] === gameArray[3] &&
+      gameArray[0] === gameArray[6] &&
+      gameArray[0] !== ""
+    ) {
+      setWinMessage(`${gameArray[0]} has won!`);
+    } else if (
+      gameArray[1] === gameArray[4] &&
+      gameArray[1] === gameArray[7] &&
+      gameArray[1] !== ""
+    ) {
+      setWinMessage(`${gameArray[1]} has won!`);
+    } else if (
+      gameArray[2] === gameArray[5] &&
+      gameArray[2] === gameArray[8] &&
+      gameArray[2] !== ""
+    ) {
+      setWinMessage(`${gameArray[2]} has won!`);
+    } else if (
+      gameArray[0] === gameArray[4] &&
+      gameArray[0] === gameArray[8] &&
+      gameArray[0] !== ""
+    ) {
+      setWinMessage(`${gameArray[0]} has won!`);
+    } else if (
+      gameArray[2] === gameArray[4] &&
+      gameArray[2] === gameArray[6] &&
+      gameArray[2] !== ""
+    ) {
+      setWinMessage(`${gameArray[2]} has won!`);
+    }
+  };
+
+  // Draw
+  const checkDraw = () => {
+    gameArray.forEach((pos) => {
+      if (pos === "cross" || pos === "circle") {
+        count++;
+      }
+      if (count === 9) {
+        setWinMessage("Game Draw!");
+      }
+    });
+    <PlayAgain />;
+  };
+
+  // Card on click
+  const changeItem = (index) => {
+    if (winMessage) {
+      return toast("Game is already finished!", { type: "success" });
+    }
+
+    if (gameArray[index] === "") {
+      gameArray[index] = isCross ? "cross" : "circle";
+      setIsCross(!isCross);
+    } else {
+      return toast("This place is already occupied!", { type: "error" });
+    }
+    findWinner();
+    checkDraw();
+  };
+
+  return (
+    <>
+      <h2 className="text-center">Choose Your Side</h2>
+      <section className="buttonGroup">
+        <button className="gameButton" onClick={() => setIsCross(true)}>
+          <FaTimes className="icon" />
+        </button>
+        <button className="gameButton" onClick={() => setIsCross(false)}>
+          <FaRegCircle className="icon" />
+        </button>
+      </section>
+
+      <section className="center">
+        <ToastContainer position="bottom-center"> </ToastContainer>
+        <main>
+          <div>
+            {winMessage ? (
+              <div>
+                <h3 className="text-center">{winMessage}</h3>
+                <PlayAgain />

```

Comment:
Here you will have `<div className="center"><PlayAgainButton onClick={onPlayAgainClick}></div>`